/**:
  ros__parameters:
    racing_simulator:
      dt: 0.01
      repeat_state_dt: 5.0
      publish_tf: true
      visualize_boundary: true
      visualize_abscissa: true
      visualize_vehicle: true
      use_frenet: false
      x0: [-24.5, 39.4, -1.0, 3.0, 0.0, 0.0]
      # x0: [0.0, 1.0, 0.1, 10.0, 0.0, 0.0]
      step_mode: continuous # [continuous, step]
